bedrock/base/templates/base-resp.html

```diff
@@ -7,6 +7,9 @@
   <head>
     <meta charset="utf-8">{# Note: Must be within first 512 bytes of page #}
 
+    {% javascript 'site' %}
+    {% block newrelic %}{% endblock %}
```

Comment:
Out of curiosity, any reason to move the JS up in the `<head>`? Does New Relic need to come in this early?